pkg/smartcontract/parameter.go

```diff
@@ -25,6 +25,11 @@ type Parameter struct {
 	Value any `json:"value"`
 }
 
+// Convertible is something that can be converted to Parameter.
+type Convertible interface {
+	ToParameter() (Parameter, error)
```

Comment:
It looks nice in `smartcontract`, but it'll be implemented by other packages where it's better be `ToSCParameter` (otherwise it's unobvious what `parameter` is meant).

Reply:
Sure, it's even in the original issue, but somehow I've skipped this. Maybe `ToSmartContractParameter` to match `stackitem.Convertible` (we have `ToStackItem` there)?

Reply:
OK, tried, but looks ugly, let's go with `ToSCParameter`.

Reply:
Fixed.